maven-spy/src/main/java/org/jenkinsci/plugins/pipeline/maven/eventspy/reporter/FileMavenEventReporter.java

```diff
@@ -85,6 +87,7 @@ public FileMavenEventReporter() throws IOException {
         } catch (IOException e) {
             throw new RuntimeIOException(e);
         }
+        isOpen = true;
```

Comment:
why do you need this `isOpen` flag? It's the only constructor, I don't understand how `isOpe`  could be false.